README.md

```diff
@@ -6,34 +6,36 @@ nion is heavily inspired by [Apollo](http://www.apollodata.com/) and [GraphQL](h
 
 ## In a Nutshell 🌰
 
-nion is used as a **decorator function** which declares what data will be managed by the decorated component and passes in props for managing that data.
+nion is used as a **hook** which is given a declaration of what data is needed by the component that calls it.
 
 ```javascript
-@nion({
-    currentUser: {
+import { exists, useNion } from 'nion'
```

Comment:
I reckon we can ditch `exists` in this example, we don't really use it much anymore anyhow 👍